tests/suites/0_stateless/02_function/02_0048_function_semi_structureds_parse_json.result

```diff
@@ -0,0 +1,45 @@
+NULL
+true
+123
+12.34
+null
+true
+-17
+123.12
+191.2
+"Om ara pa ca na dhih"
+[-1,12,289,2188,false]
+{"x":"abc","y":false,"z":10}
+ERROR 1105 (HY000) at line 13: Code: 1010, displayText = Error parsing JSON: Error("EOF while parsing a value", line: 1, column: 3).
```

Comment:
Please use the error-code in *.sql to skip the error instead writing them to the result.

Reply:
ok